paasta_tools/frameworks/native_scheduler.py

```diff
@@ -190,6 +200,18 @@ def get_old_tasks(self, name, tasks):
     def is_task_new(self, name, tid):
         return tid.startswith("%s." % name)
 
+    def killer_for_task(self, driver, taskId):
+        """Returns a closure to kill a task
+        """
+        def kill_task():
+            kill_task(driver, taskId)
```

Comment:
this confuses me because it looks like recursion. is there anything wrong with using lambda on line 213?

    return Timer(timeout_value, lambda: self.kill_task(driver, taskId))